MAINTAINERS.md

```diff
@@ -0,0 +1,61 @@
+# Maintainers
+
+## The maintainer list of all repositories(names are in alphabetical order)
+* Chunlin Yang (Redhat, @clyang82)
+* DONG BEIQING (eBay, @dongbeiqing)
+* Hao Lin (eBay, @haowells)
+* Jian Qiu (Redhat, @qiujian16)
+* Jian Zhu (Redhat, @zhujian7)
+* Le Yang (Redhat, @elgnay)
+* LongLong Cao (Redhat, @morvencao)
+* Mike Ng (Redhat, @mikeshng)
+* Qing Hao (Redhat, @haoqing0110)
+* Zhao Xue (Redhat, @xuezhaojun)
+* Zhiwei Yin (Redhat, @zhiweiyin318)
+
+## The maintainers in different areas of OCM
+
+### [OCM CORE](https://github.com/open-cluster-management-io/ocm)
+* General maintainers:
+    * Jian Qiu
+* Maintainers for specific areas:
+    * `registration`
+        * Le Yang
+        * Wei Liu
+        * Zhao Xue
+    * `work`
+        * Le Yang
+        * Jian Zhu
+    * `placement`
+        * Le Yang
+        * Qing Hao
+    * `operator`
+        * Jian Zhu
+        * Zhiwei Yin
+        * DONG BEIQING
+    * [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane)
+        * Chunlin Yang
+
+### API, SDK and Tools
+* [addon-framework](https://github.com/open-cluster-management-io/addon-framework)
+    * Jian Qiu
+    * Le Yang
+    * Zhiwei Yin
+    * Jian Zhu
+* [sdk-go](https://github.com/open-cluster-management-io/sdk-go/blob/main/OWNERS)
+    * Jian Qiu
```

Comment:
we cannot have only one maintainer for this.

Reply:
Another name in OWNERS file is David, should I put David in the maintainer of sdk-go?

Reply:
we might need to tweak the sdk-go owner file